websites/management/commands/update_departments.py

```diff
@@ -0,0 +1,33 @@
+from functools import reduce
```

Comment:
You can delete this command now that there is a migration to accomplish the same thing.